google-oauth-client-jetty/src/main/java/com/google/api/client/extensions/jetty/auth/oauth2/LocalServerReceiver.java

```diff
@@ -115,28 +133,42 @@ public LocalServerReceiver() {
 
   @Override
   public String getRedirectUri() throws IOException {
-    server = new Server(port != -1 ? port : 0);
-    Connector connector = server.getConnectors()[0];
-    connector.setHost(host);
-    server.setHandler(new CallbackHandler());
+
+    server = HttpServer.create(new InetSocketAddress(port != -1 ? port : findOpenPort()), 0);
+    HttpContext context = server.createContext(callbackPath, new CallbackHandler());
+    server.setExecutor(null);
+
     try {
       server.start();
-      port = connector.getLocalPort();
+      port = server.getAddress().getPort();
     } catch (Exception e) {
       Throwables.propagateIfPossible(e);
       throw new IOException(e);
     }
-    return "http://" + connector.getHost() + ":" + port + callbackPath;
+    return "http://" + this.getHost() + ":" + port + callbackPath;
+  }
+
+  /*
+   *Copied from Jetty findFreePort() as referenced by: https://gist.github.com/vorburger/3429822
```

Comment:
It might be the Javadoc comment that checkstyle is complaining about; I'm not sure. Try deleting it.